RallyHereDeveloperAPI/Source/RallyHereDeveloperAPI/Public/DevRole.h

```diff
@@ -227,11 +227,11 @@ struct RALLYHEREDEVELOPERAPI_API FRHAPI_DevRole : public FRHAPI_DevModel
     /** @brief Sets the value of RoleId_Optional and also sets RoleId_IsSet to true */
     void SetRoleId(FGuid NewValue) { RoleId_Optional = NewValue; RoleId_IsSet = true; }
      /** @brief Clears the value of RoleId_Optional and sets RoleId_IsSet to false */
-    void ClearRoleId() { RoleId_Optional = FGuid(TEXT("f698515f-c38d-40bc-944f-68e0e1228cff")); RoleId_IsSet = false; }
+    void ClearRoleId() { RoleId_Optional = FGuid(TEXT("92044d78-7b86-469d-8d60-dec5e7435e75")); RoleId_IsSet = false; }
```

Comment:
Where are these guids coming from?

Reply:
Oh, these are default values from the API correct?

Reply:
Yeah, this was defaults made from the API.  Not sure why the default was changed though